ovos_plugin_common_play/ocp/player.py

```diff
@@ -492,16 +505,23 @@ def play_next(self):
         self.play()
 
     def play_prev(self):
+        """
+        Play the previous track in the playlist.
+        If there is no previous track, do nothing.
+        """
         if self.active_backend in [PlaybackType.MPRIS]:
             if self.mpris:
                 self.mpris.play_prev()
             return
-        elif self.active_backend in [PlaybackType.SKILL, PlaybackType.UNDEFINED]:
-            self.bus.emit(Message(f'ovos.common_play.{self.now_playing.skill_id}.prev'))
+        elif self.active_backend in [PlaybackType.SKILL,
+                                     PlaybackType.UNDEFINED]:
+            self.bus.emit(Message(
+                f'ovos.common_play.{self.now_playing.skill_id}.prev'))
             return
         self.pause()  # make more responsive
 
         if self.shuffle:
+            # TODO: Should skipping back get a random track instead of previous?
```

Comment:
in shuffle mode i think next usually means you didnt like the song but if we ask for previous is because we want to repeat the same song, eg, to click the like button (once we add it)

Reply:
That's what I was thinking, so going back should not do anything different if `self.shuffle == True`

Reply:
can you do that change in a follow up PR and keep this one just with all the awesome tests and docstrs for existing behavior?

Reply:
I'll just leave that TODO in then